Mathlib/Algebra/BigOperators/Group/Finset/Defs.lean

```diff
@@ -170,9 +170,9 @@ def bigOpBindersProd (processed : Array (Term × Term)) : MacroM Term := do
 
 These support destructuring, for example `∑ ⟨x, y⟩ ∈ s ×ˢ t, f x y`.
 
-Notation: `"∑" bigOpBinders* ("with" (ident ":")? term)? "," term` -/
+Notation: `"∑" bigOpBinders* (" with" (ident ":")? term)? "," term` -/
 syntax (name := bigsum)
-  "∑ " bigOpBinders ("with " atomic(binderIdent " : ")? term)? ", " term:67 : term
+  "∑ " bigOpBinders (" with " atomic(binderIdent " : ")? term)? ", " term:67 : term
```

Comment:
```suggestion
  "∑ " bigOpBinders (" with " atomic(binderIdent ": ")? term)? ", " term:67 : term
```
Btw, this space is probably superfluous, but I cannot test right now.

Reply:
I don't think that it is a bug: that space there is superfluous, since the ` with ` already implies it.

Anyway, adding the other space that you added is the important bit!

Reply:
The bug I'm referring to is how the delaborator has not been implemented to turn `dite` expressions to `with`.